presto-docs/src/main/sphinx/sql/create-function.rst

```diff
@@ -7,7 +7,7 @@ Synopsis
 
 .. code-block:: none
 
-    CREATE [ OR REPLACE ] FUNCTION
+    CREATE [ OR REPLACE ] [TEMPORARY] FUNCTION
```

Comment:
From the logic, it's not clear if we are supporting replace.

Reply:
The analyzer will throw if REPLACE is used with TEMPORARY, and there is a test for it in TestAnalyzer.